pkgs/development/python-modules/colout/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, babel
+, buildPythonPackage
+, fetchPypi
+, pygments
+}:
+
+buildPythonPackage rec {
+  pname = "colout";
+  version = "0.12.1";
+  format = "wheel";
```

Comment:
Is there no way to build it from source?

Reply:
obviously no => https://pypi.org/project/colout/#files

Reply:
PyPI do not have sometimes the records for source, but GitHub seems to have it here: https://github.com/nojhan/colout/blob/master/setup.py

Reply:
(otherwise, we have to mark the provenance of this package as binary-only which would be a shame)